artemis-journal/src/main/java/org/apache/activemq/artemis/core/io/buffer/TimedBuffer.java

```diff
@@ -93,10 +82,10 @@
    public TimedBuffer(final int size, final int timeout, final boolean logRates) {
       bufferSize = size;
 
-      this.logRates = logRates;
-
       if (logRates) {
```

Comment:
NIT: is there a better name for this, i can't think of one, but name just seems misleading a little, as actually this is enabling a profiler not logging out anything to the std log file.